File: benchmarks/src/main/resources/python/fibonacci_recursive_starter.py
import sys
import time
from fibonacci_recursive import fibonacci

n = int(sys.argv[1])
times = []
for run in range(1, 11):
  start = time.time()
  fib = fibonacci(n)
  executionTime = time.time() - start
  print("Run %d: %2.5f ms" % (run, executionTime))
  times.append(executionTime)

print("Average: %2.5f ms\n" % (sum(times) / len(times)))

File: benchmarks/src/main/resources/js/fibonacci.recursive.starter.js
for (var run = 1; run <= 10; ++run) {
  console.time('fibonacci');
  fibonacci(40);
  console.timeEnd('fibonacci');
}

function fibonacci(n) {
  if (n <= 1) {
    return n;
  }

  var nMinusOne = fibonacci(n - 1);
  var nMinusTwo = fibonacci(n - 2);

  return nMinusOne + nMinusTwo;
}